### ISSUE TITLE
Immediate exception after windows inside upgrade

### ISSUE THREAD

**Issue by mucki-at** [PowerShell/PSReadLine#3005]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
Oops, something went wrong.
Please report this bug with ALL the details below, including both the 'Environment' and 'Exception' sections.
Please report on GitHub: https://github.com/PowerShell/PSReadLine/issues/new?template=Bug_Report.md
Thank you!

### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22498
BufferWidth: 120
BufferHeight: 30

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

![image](https://user-images.githubusercontent.com/23323005/141171244-d4be6123-eeda-4b18-8250-cb2b179e2a2c.png)


### Environment data

```console
cannot complete. Neither pasting, nor trying to store as PS1 and executing produces any output except more exceptions.
```

### Steps to reproduce

NB: I tried to repro on PSReadLine 2.1.0 but i cannot seem to downgrade from 2.2.0-beta1 (I don't know how i ended up with the beta version either).

1. Upgrade to insider build 22498.rs_onecore_sigma_rd.211105-1700
2. Start powershell

### Expected behavior

nothing, just powershell prompt

### Actual behavior

immediate exception, even though some commands can still be run (and seem to work), but output is spammed with exception reports.

**Comment by jonathan-b-wiebe** [comment]
Confirming the above behavior on Windows 10.0.19043 after upgrading from Powershell 7.1.5 to 7.2.0

Only differences from the above bug report are OS version and the Buffer height (mine in 9001 vs the 30 reported above).

**Comment by emil-khusnutdinov** [comment]
OS: Microsoft Windows 10.0.19043
After upgrading from PowerShell 7.1.5 to 7.2.0

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
